shuup/core/basket/objects.py

```diff
@@ -52,7 +52,13 @@ def cache_info(self, request):
         self.gross_weight = product.gross_weight
         self.shipping_mode = product.shipping_mode
         self.sku = product.sku
-        self.text = product.safe_translation_getter("name", any_language=True)
+        self.text = ""
+
+        shop_product = self.shop_product
```

Comment:
if you check the property above, the shop_product has the following code:
```
return self.product.get_shop_instance(self.shop)
```
It is not that expensive, but.. :)